### PR TITLE
Update Tracking

### PR DESCRIPTION
**Pull request by federicoweber**
### Description
To update tracking events according to this [proposal](https://threads.com/34397631629)

### Context & Notes
- it's using `BufferTracker` to ensure consistent data tracking'
- all events now include the `organizationId`
- Publish App is not sending the `identify` event anymore, and it will rely on the `AppShell` for that

~This is still a WIP as this is using the `analytics-middleware` exported by Analyze we will need to merge https://github.com/bufferapp/buffer-analyze/pull/1072 and update the middleware version before shipping this.~
Publish is using it's own version of the `analytics-tracker`, so the change to use `BufferTracker`, is out of the scope of this PR, will leave it to the team to update it when it's convenient 🙂 

### Best Practices Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] I have tested different user stories.
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

### ⚠️ Pre-deploy Checklist

- [ ] I have manually loaded the URL of the staging deployment and confirmed it's working (see below**)
- [ ] I have confirmed that the Cypress E2E test is passing (_Note: we have a known issue where it doesn't run on some PRs. Hopefully this will be fixed soon._)
- [ ] I have confirmed that all other tests are passing.

** To visit the URL of the staging deployment, please vlick the link in the bot comment, or look under checks at the bottom of this PR and click "Details" next to `bufferbotbrains/cicd-buffer-publish`
image

### CODE REVIEW

File: webpack.config.dev.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const merged = merge.strategy({
     disableHostCheck: true,
     hot: true,
     port: 8888,
-    host: 'publish.local.buffer.com',
+    host: '0.0.0.0',
```
Comment (karelm) [inline webpack.config.dev.js:23]: Heya @federicoweber, how come you've changed the bindings to attach to 0.0.0.0 here? (just curious) Is it needed to update the tracking? Is it because of some side-effects we weren't aware of? I'd love to understand why this change is necessary

**Review by karelm — APPROVED** [review]
I've left a comment around the change of host. I don't think its a blocker so feel free to merge this in if necessary, but I am curious about the change

File: webpack.config.dev.js (hunk 1 above)
Comment (federicoweber) [reply webpack.config.dev.js:23]: hey @karelm oh sorry I commited that by mistake 😅. 
So the `0.0.0.0` will always resolve to the machine running the service, I have all the local env. I work with set to `0.0.0.0` as they will still resolve fine with the local URL `*.local.buffer.com`, so they won't effect the workflow for anyone else, but it will allow me to run those properly from my VM setup :). Will revert that.

**Comment by federicoweber** [comment]
Thanks for the review @karelm 🌮
